resources/js/app.js

```diff
@@ -0,0 +1,282 @@
+  cardTop.append(cardHeader, cardList);
+  const cardImage = document.createElement('img');
+  cardImage.classList.add('cardimage');
+  const cardText = document.createElement('p');
+  cardText.classList.add('works__paragraph', 'popup-paragraph');
+  const cardWorksList = document.createElement('ul');
+  cardWorksList.classList.add('cardworkslist');
+  const cardButtonDiv = document.createElement('div');
+  cardButtonDiv.classList.add('cardbuttondiv');
+  const cardButtonLive = document.createElement('a');
+  cardButtonLive.classList.add('btn-live');
+  cardButtonLive.innerHTML = 'See Live';
+  const cardButtonImg = document.createElement('img');
+  cardButtonImg.classList.add('livebtnimg');
+  cardButtonImg.setAttribute('src', 'resources/img/export.svg');
+  cardButtonLive.appendChild(cardButtonImg);
+  const cardButtonSource = document.createElement('a');
+  cardButtonSource.classList.add('btn-source');
+  cardButtonSource.innerHTML = 'See Source';
+  const cardButtonImg2 = document.createElement('img');
+  cardButtonImg2.classList.add('sourcebtnimg');
+  cardButtonImg2.setAttribute('src', 'resources/img/gitbutton.svg');
+  cardButtonSource.appendChild(cardButtonImg2);
+  const cardWorksButtons = document.createElement('div');
+  cardWorksButtons.classList.add('cardworksbuttons');
+  cardWorksButtons.append(cardWorksList, cardButtonDiv);
+  const xContainer = document.createElement('div');
+  const cardXButton = document.createElement('a');
+  cardXButton.innerHTML = '&times;';
+  cardXButton.classList.add('close');
+
+  cardPopup.append(
+    cardTop,
+    xContainer,
+    cardXButton,
+    cardImage,
+    cardText,
+    cardWorksButtons,
+  );
+
+  xContainer.appendChild(cardXButton);
+  cardPopupOverlay.appendChild(cardPopup);
+  cardButtonDiv.append(cardButtonLive, cardButtonSource);
+  body.appendChild(cardPopupOverlay);
+
+  cardHeader.textContent = worksInfo[num].title;
+  cardImage.src = worksInfo[num].image;
+  cardText.textContent = worksInfo[num].paragraph;
+  cardButtonLive.href = worksInfo[num].urlLive;
+  cardButtonSource.href = worksInfo[num].urlSource;
+
+  let childNode;
+  for (let i = 0; i < worksInfo[num].list.length; i += 1) {
+    childNode = document.createElement('li');
+    childNode.innerHTML = worksInfo[num].list[i];
+    cardList.append(childNode);
+  }
+  for (let i = 0; i < worksInfo[num].languages.length; i += 1) {
+    childNode = document.createElement('li');
+    childNode.innerHTML = worksInfo[num].languages[i];
+    cardWorksList.append(childNode);
+  }
+
+  const closePopup = document.querySelector('.close');
+
+  closePopup.addEventListener('click', () => {
+    body.removeChild(cardPopupOverlay);
+  });
+}
+
+openPopup.addEventListener('click', () => {
+  openBtn(0);
+});
+
+openPopup2.addEventListener('click', () => {
+  openBtn(1);
+});
+
+openPopup3.addEventListener('click', () => {
+  openBtn(2);
+});
+
+openPopup4.addEventListener('click', () => {
+  openBtn(3);
+});
```

Comment:
- [ ] [OPTIONAL] For cleaner code, kindly consider using a loop to target these buttons instead of hard coding and repeating them